controller/control.go

```diff
@@ -234,6 +234,11 @@ func (c *Controller) Start(addresses ...string) error {
 
 func (c *Controller) WriteAt(b []byte, off int64) (int, error) {
 	c.RLock()
+	if off < 0 || off+int64(len(b)) > c.size {
```

Comment:
hold on, no, why `>=`? 
Reading 100 bytes at offset 0 is fine for a 100 byte volume